### PR TITLE
Make enums for draw and modify interactions public

### PR DESCRIPTION
**Pull request by gberaudo**
Project code compiled together with OpenLayers code by the closure compiler relies on these enums.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,7 +97,6 @@
         }
       ],
       "no-constant-condition": 0,
-      "openlayers-internal/enum": 2,
```
Comment (tschaub) [inline package.json]: Instead of disabling this rule, you can follow it by giving public enums their own file.

**Comment by gberaudo** [comment]
OK. @tschaub, PR updated, please review.

**Comment by tschaub** [comment]
Looks like the rule still needs to be re-enabled.

**Comment by gberaudo** [comment]
Sorry I overlooked it. @tsauerwein, the rule is now restored.

**Comment by fredj** [comment]
@tschaub good to merge ?

**Pull request merged**